CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# How to contribute to the State of Rust survey
+
+Great that you want to help with the State of Rust survey! The State of Rust survey is the annual platform for us to look at how was the past year of work and for the community to express their wishes.
+
+The survey platform of choice is currently [Survey Hero](https://www.surveyhero.com). We have reached a general consensus that, while not perfect, it works rather well for us therefore we are not looking into alternatives for the time being.
+
+The State of Rust survey is published usually in the first half of December. Contributions are tipically needed once a year as we approach to the deadline. The initial work to kickoff the survey is taken care of by the [Survey Team](https://github.com/rust-lang/team/blob/master/teams/community-survey.toml). Contributions are welcome to translate the survey into multiple languages. The more, the merrier.
+
+Here's a rough timeline of the work:
+- The State of Rust survey is discussed on Zulip (in [#t-community/survey](https://rust-lang.zulipchat.com/#narrow/channel/402479-t-community.2Frust-survey)). We gather feedback about the previous year's survey, we ask the main teams if they have specific wishes and we draft the English survey
+- In the meantime we start recording interest from volunteers to translate the survey into different languages
+- After the English survey is finalized and uploaded on Survey Hero, translators can start their work
+
+Here's how the translation effort is coordinated:
```

Comment:
I'm not sure if we want to use this workflow. A large part of the help provided by the translators is that they have actually performed the translation directly in SurveyHero, which was quite nice. Especially for `zh-CN`, I couldn't even replicate the SH contents by copy-pasting from Markdown, lol.

On the other hand, reviewing the changes on GitHub is *much* easier, and multiple people can coordinate. And now that we actually have decent tooling, and hopefully also a reasonably good translation baseline on SH, it might not be so bad to just do everything on GH, and then do one final manual sync to SH. It was sure quite messy this year, when the changes were made to both SH and MD in parallel, and we had to frequently resync in both directions.

Reply:
You're right, the workflow I described is in part wrong (the first step is the contributor working on SH). So for now I have amended the description to reflect this years' workflow because it has happened (please check if now is correct).

How do you prefer to work next year? GH only? Or like this year? Or do we want to discuss this at a later time, after we're done?

Reply:
The current description looks good. Let's discuss the rest next year :laughing: